core/server/data/migrations/versions/4.35/2022-02-01-11-48-update-email-recipient-filter-column-type.js

```diff
@@ -0,0 +1,25 @@
+const logging = require('@tryghost/logging');
+const {createNonTransactionalMigration} = require('../../utils');
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
+        if (knex.client.config.client === 'sqlite3') {
+            logging.warn('Skipping migration for SQLite3');
+            return;
+        }
+        logging.info('Changing posts.email_recipient_filter column from VARCHAR(50) to TEXT');
+        await knex.schema.alterTable('posts', function (table) {
+            table.text('email_recipient_filter').alter();
+        });
+    },
+    async function down(knex) {
+        if (knex.client.config.client === 'sqlite3') {
+            logging.warn('Skipping migration for SQLite3');
+            return;
+        }
+        logging.info('Changing posts.email_recipient_filter column from TEXT to VARCHAR(50)');
+        knex.schema.alterTable('posts', function (table) {
+            table.string('email_recipient_filter', 50).alter();
```

Comment:
These `down()` operations will fail if content is longer than 50 chars:
```mysql
mysql> alter table posts modify email_recipient_filter varchar(50);
ERROR 1406 (22001): Data too long for column 'email_recipient_filter' at row 1
```

Reply:
Smart! What shall we do? noop on `down` I guess?

Reply:
Done